futurenhs.automation/pageObjects/basePage.js

```diff
@@ -1,3 +1,4 @@
+const { getLeadingCommentRanges } = require('typescript');
```

Comment:
Where are you using this?

Reply:
Good spot! Think this is intellisense at play again... Does this from time to time. I'll resolve and push the change up.